init.sh

```diff
@@ -12,13 +12,16 @@ decentrcli config keyring-backend test
 decentrcli keys add jack
 decentrcli keys add alice
 
-decentrd add-genesis-account $(decentrcli keys show jack -a) 1000000udec
-decentrd add-genesis-account $(decentrcli keys show alice -a) 1000000udec
+decentrd add-genesis-account $(decentrcli keys show jack -a) 1000000000udec
+decentrd add-genesis-account $(decentrcli keys show alice -a) 1000000000udec
 decentrd add-genesis-community-moderators $(decentrcli keys show jack -a)
 decentrd add-genesis-pdv-cerberuses $(decentrcli keys show jack -a)
 
 decentrd gentx --name jack --keyring-backend test --amount 1000000udec
 
+# change genesis gov denom
```

Comment:
Would it be possible to implement with a code similar to 
```golang
// stakingAppModuleDecorator is staking app module decorator to replace the default bond denom
// "stake" with "dec".
type stakingAppModuleDecorator struct {
	staking.AppModule
}

func NewStakingAppModuleDecorator(keeper staking.Keeper, accountKeeper auth.AccountKeeper, supplyKeeper supply.Keeper) *stakingAppModuleDecorator {
	return &stakingAppModuleDecorator{staking.NewAppModule(keeper, accountKeeper, supplyKeeper)}
}

func (a stakingAppModuleDecorator) DefaultGenesis() json.RawMessage {
	params := staking.DefaultParams()
	params.BondDenom = DefaultBondDenom

	return staking.ModuleCdc.MustMarshalJSON(staking.GenesisState{
		Params: params,
	})
}
```
?

Reply:
It's not possible, the gov module uses const(`stake`) for bond denom in `InitGenesis` method.

we can't specify it even with ldflags